src/private_broadcast.h

```diff
@@ -7,17 +7,23 @@
 
 #include <net.h>
 #include <primitives/transaction.h>
+#include <primitives/transaction_identifier.h>
 #include <sync.h>
 #include <threadsafety.h>
 #include <util/time.h>
 
 #include <optional>
+#include <tuple>
 #include <unordered_map>
 #include <vector>
 
 /**
  * Store a list of transactions to be broadcast privately. Supports the following operations:
  * - Add a new transaction
+ * - Pick a transaction for sending to one recipient
+ * - Query which transaction has been picked for sending to a given recipient node
+ * - Mark that a confirmation of reception for a transaction by a given recipient node
```

Comment:
a57ef7ddbacebf23c4c50c3fc58615a58339dcc2

nit: suggestion
`- Mark that given recipient node has confirmed receipt of a transaction`